### DISCUSSION TITLE
Complete revamp of GE/Jasco 0x0063

### DISCUSSION THREAD

**Discussion by disforw** [zwave-js/zwave-js#4620] [General] [answered]
Would you allow me to open a PR and clean up every GE/Jasco device, moving all config parameters to template?

Over the years of automating my house, I now have 9 different GE/Jasco models installed around the house. Having mismatching config language is making automations more difficult. I’m willing to put in the time and do the work if you will accept it.

**Comment by AlCalzone** [comment, marked as answer]
I would appreciate it very much!
If possible, please open a draft PR early, so we can review and discuss some of the changes with @blhoward2 before you are in too deep.

**Reply by disforw** [reply]
PR #4622
